packages/app/src/messages/components/messages-input-search/messages-input-search.helper.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const searchResultFormatter = (
```

Comment:
Just wondering - have you considered implementing `searchResultFormatter` just as a React component or just leaving it as a function that returns `JSX` elements? By that you would avoid using `dangerouslySetInnerHTML` (not 100% sure, but maybe there is a chance that searching would break the rendered HTML).